app/main/views/briefs.py

```diff
@@ -59,16 +59,16 @@ def ask_brief_clarification_question(brief_id):
 
 @main.route('/opportunities/<int:brief_id>/responses/create', methods=['GET'])
 @login_required
-def submit_brief_response(brief_id):
+def brief_response(brief_id):
```

Comment:
I started with `create_` but then checked our code elsewhere and our most usual pattern, e.g. in https://github.com/alphagov/digitalmarketplace-supplier-frontend/blob/master/app/main/views/suppliers.py, seems to be to have the `name_of_the_thing()` on the GET route and `submit_name_of_the_thing()` on the POST route everywhere: